clang/lib/Headers/wasm_simd128.h

```diff
@@ -1888,18 +1888,15 @@ static __inline__ v128_t __FP16_FN_ATTRS wasm_f16x8_splat(float __a) {
   return (v128_t)__builtin_wasm_splat_f16x8(__a);
 }
 
-static __inline__ float __FP16_FN_ATTRS wasm_f16x8_extract_lane(v128_t __a,
-                                                                int __i)
-    __REQUIRE_CONSTANT(__i) {
```

Comment:
out of curiosity, what does this REQUIRE_CONSTANT actually do?

Reply:
It does require a constant in C code, but in a no-opt build it is not a constant in LLVM IR.